### DISCUSSION TITLE
Any plans to promote Roc as the extensions language of choice

### DISCUSSION THREAD

**Discussion by favetelinguis** [zed-industries/zed#13941] [Feature Requests]
Given the inventor of Roc is in the team and that a Zed platform in Roc to compile WASM sounds very interesting. Are there any plans to promote Roc as the No1 extensions language? Having a dominating language looks to be the natural evolution looking at nvim where you can use multiple languages but still Lua is what everyone use.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
